### PR TITLE
Readme update

### PR DESCRIPTION
**Pull request by CasperMok**
Readme update 
* tech-pub update
* Sample code API naming & demo stream name

### CODE REVIEW

**Review by aravind-raveendran — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -10,22 +10,23 @@ Adding shields would also be amazing -->
...
-> **_Info:_** There are two parties in RTS - a publisher and a viewer. A publisher is one who broadcasts the stream. A viewer(monitor) who consumes the stream. This UIKit is meant for viewer/monitor applications. Please refer to this [blog post](https://dolby.io/blog/real-time-streaming-with-dolby-io/) to understand the ecosystem.
+> **_Note:_** There are two parties in RTS - a publisher and a viewer. A publisher is one who broadcasts the stream. A viewer is one who consumes the stream. This UIKit is meant for viewer/monitor applications. Please refer to this [blog post](https://dolby.io/blog/real-time-streaming-with-dolby-io/) to understand the ecosystem.
 
 ## Requirements
 
 This setup guide is validated on both Intel and Apple Silicon based MacBook Pro machines running macOS 13.4.
 
 * Xcode Version 14.3.1 (14E300c)
 * iPhone device or simulator running iOS 15.0
+* Knowledges on iOS, Swift and SwiftUI programming
```
Comment (dubeyShivank) [inline README.md:29]: Replace with :
` Familiarity with iOS development using Swift and Swift UI`

Comment (CasperMok) [reply README.md:29]: Update on PR, https://github.com/DolbyIO/rts-uikit-ios/pull/21

File: README.md [hunk 2]
```diff
@@ -92,6 +93,8 @@ struct ContentView_Previews: PreviewProvider {
 }
 ```
 
+This app has a `Start Stream` button which starts monitoring a demo stream in a streaming screen. This streaming screen provided by this kit is self contained. It has the necessary  components for monitoring a streaming. It has a settings menu, in which it provides different video views, audio selections, and streaming sorting orders.
```
Comment (dubeyShivank) [inline README.md:96]: Double blank space in this bit

`It has the necessary  components`

Comment (dubeyShivank) [reply README.md:96]: > This app has a `Start Stream` button which starts monitoring a demo stream in a streaming screen. This streaming screen provided by this kit is self contained. It has the necessary  components for monitoring a streaming. It has a settings menu, in which it provides different video views, audio selections, and streaming sorting orders.

to 

This app has a `Start stream` button which starts playing a demo stream in the app. The streaming screen contains all the components you need in order to build your own experience - including a settings menu where you can select different video layouts, audio source selection and stream sort order.

**Pull request merged**